samples/msgext-doc-search-csharp/Search/SearchApp.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Teams;
+using Microsoft.Bot.Schema;
+using Microsoft.Bot.Schema.Teams;
+using AdaptiveCards;
+using Newtonsoft.Json.Linq;
+using Azure.Search.Documents.Models;
+
+namespace msgext_doc_search_csharp.Search;
+
+public class SearchApp : TeamsActivityHandler
+{
+    private readonly string _adaptiveCardFilePath = Path.Combine(".", "Resources", "helloWorldCard.json");
+    private readonly AISearch _aiSearch;
+
+    public SearchApp(AISearch aiSearch)
+    {
+        _aiSearch = aiSearch;
+    }
+
+    // Search
+    protected override async Task<MessagingExtensionResponse> OnTeamsMessagingExtensionQueryAsync(ITurnContext<IInvokeActivity> turnContext, MessagingExtensionQuery query, CancellationToken cancellationToken)
+    {
+        var templateJson = await System.IO.File.ReadAllTextAsync(_adaptiveCardFilePath, cancellationToken);
+        var template = new AdaptiveCards.Templating.AdaptiveCardTemplate(templateJson);
+
+        var text = query?.Parameters?[0]?.Value as string ?? string.Empty;
+        var hybridResponse = await _aiSearch.SemanticHybridSearch(text);
+        var attachments = new List<MessagingExtensionAttachment>();
+        Console.WriteLine($"Semantic Hybrid Search Results:");
```

Comment:
Let's change these `Console.WriteLine` statements to use `Debug.WriteLine` instead.